packages/dtslint/package.json

```diff
@@ -40,6 +40,7 @@
     "yargs": "^15.1.0"
   },
   "peerDependencies": {
+    "@arethetypeswrong/cli": "^0.13.1",
```

Comment:
Should this just be a regular dep? Theoretically this _could_ cause errors to people installing the tool as some package managers don't auto install peer deps (I think we configured pnpm to do so).

Reply:
I added it as a dep to DT so the attw version and config would be managed in the same place. That doesn’t buy us _that_ much though, so  I don’t mind changing it.